### PR TITLE
debian fixes

### PR DESCRIPTION
**Pull request by ashish-jabble**
@oshadmon Please test the attached debian.

[foglamp-south-wind-turbine.zip](https://github.com/foglamp/foglamp-south-wind-turbine/files/3271859/foglamp-south-wind-turbine.zip)

### CODE REVIEW

**Comment by oshadmon** [comment]
On both Ubuntu 16.04 & 18.04 the install script fails due to a premmissions in regard to `./usr/local/foglamp/python/requirements-wind_turbine.sh`

Please note when manually running the shell script (`sudo bash /var/lib/dpkg/info/foglamp-south-wind-turbine.postinst: ./usr/local/foglamp/python/requirements-wind_turbine.sh`) than the installations occurs. 

```
foglamp-1.6.0-x86_64_ubuntu_16_04  foglamp-1.6.0-x86_64_ubuntu_16_04.tgz  foglamp-south-wind-turbine-1.6.0-amd64.deb  foglamp-south-wind-turbine.zip
ubuntu@ori-ubuntu-1604:~$ sudo apt -y install ./foglamp-south-wind-turbine-1.6.0-amd64.deb
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Note, selecting 'foglamp-south-wind-turbine' instead of './foglamp-south-wind-turbine-1.6.0-amd64.deb'
The following NEW packages will be installed:
  foglamp-south-wind-turbine
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 0 B/6,078 B of archives.
After this operation, 0 B of additional disk space will be used.
Get:1 /home/ubuntu/foglamp-south-wind-turbine-1.6.0-amd64.deb foglamp-south-wind-turbine amd64 1.6.0 [6,078 B]
Selecting previously unselected package foglamp-south-wind-turbine.
(Reading database ... 112879 files and directories currently installed.)
Preparing to unpack .../foglamp-south-wind-turbine-1.6.0-amd64.deb ...
Unpacking foglamp-south-wind-turbine (1.6.0) ...
Setting up foglamp-south-wind-turbine (1.6.0) ...
/var/lib/dpkg/info/foglamp-south-wind-turbine.postinst: 38: /var/lib/dpkg/info/foglamp-south-wind-turbine.postinst: ./usr/local/foglamp/python/requirements-wind_turbine.sh: Permission denied
dpkg: error processing package foglamp-south-wind-turbine (--configure):
 subprocess installed post-installation script returned error exit status 126
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

**Comment by ashish-jabble** [comment]
@oshadmon Permission issue has been resolved and also removed Python3 specific prerequisite in  https://github.com/foglamp/foglamp-south-wind-turbine/pull/1/commits/a01a38fd38b1aa2ce6aa95d6dcf79691c333fe22 as pip package is available for Phidget22. Kindly test the newly attached debian.

[foglamp-south-wind-turbine.zip](https://github.com/foglamp/foglamp-south-wind-turbine/files/3276530/foglamp-south-wind-turbine.zip)

File: requirements.sh [hunk 1]
```diff
@@ -41,9 +40,4 @@ fn=`find -name *libphidget22.rule*`
 sudo mv ${fn} /etc/udev/rules.d.
 
 # Python3 specific prerequisites - https://www.phidgets.com/docs/Language_-_Python 
-cd /tmp/wind-turbine-install
-wget https://www.phidgets.com/downloads/phidget22/libraries/any/Phidget22Python.zip
-unzip /tmp/wind-turbine-install/Phidget22Python.zip
-cd /tmp/wind-turbine-install/Phidget22Python
-sudo python3 setup.py install
-sudo rm -rf /tmp/wind-turbine-install
+pip3 install Phidget22
```
Comment (praveen-garg) [inline requirements.sh:43]: Better to move this to `requirements-__PLUGIN_NAME__.txt`

**Review by oshadmon — APPROVED** [review]
LGTM

File: requirements.sh (hunk 1 above)
Comment (ashish-jabble) [reply requirements.sh:43]: it will take care in FOGL-2887

**Pull request merged**